From cc61850a28855ee7778b73e5ed70c23ac26dd823 Mon Sep 17 00:00:00 2001
From: Nate Drude <nate.drude@emerson.com>
Date: Thu, 25 Jul 2019 08:56:23 -0500
Subject: [PATCH] A5-9: Fix compile issue when no opengl in
 QVideoSurfacePainterFix qtmultimedia build issue with QT 5.12.4 when
 -no-opengl is used

See: https://gitweb.gentoo.org/proj/qt.git/tree/dev-qt/qtmultimedia/files/qtmultimedia-5.12.4-fix-no-opengl.patch?id=c0f69728fe2538b50a9daf12ea241a76c4b5d131
---
 src/multimediawidgets/qpaintervideosurface.cpp | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/src/multimediawidgets/qpaintervideosurface.cpp b/src/multimediawidgets/qpaintervideosurface.cpp
index 440d5c8..3e43814 100644
--- a/src/multimediawidgets/qpaintervideosurface.cpp
+++ b/src/multimediawidgets/qpaintervideosurface.cpp
@@ -57,10 +57,6 @@
 #ifndef GL_RGB8
 #define GL_RGB8 0x8051
 #endif
-#endif
-
-#include <QtDebug>
-QT_BEGIN_NAMESPACE
 
 static void makeCurrent(QGLContext *context)
 {
@@ -71,6 +67,11 @@ static void makeCurrent(QGLContext *context)
         handle->makeCurrent(handle->surface());
 }
 
+#endif
+
+#include <QtDebug>
+QT_BEGIN_NAMESPACE
+
 QVideoSurfacePainter::~QVideoSurfacePainter()
 {
 }
